pkg/utils/conditions/condition.go

```diff
@@ -29,7 +29,7 @@ func checkAnyAllConditions(logger logr.Logger, ctx enginecontext.Interface, cond
 	return len(condition.AnyConditions) == 0, nil
 }
 
-func checkCondition(logger logr.Logger, ctx enginecontext.Interface, condition kyvernov2beta1.Condition) (bool, error) {
+func CheckCondition(logger logr.Logger, ctx enginecontext.Interface, condition kyvernov2beta1.Condition) (bool, error) {
```

Comment:
There is no need to make it public since it is only used internally in the package. You can use `checkCondition` instead of `CheckCondition`

Reply:
Fixed